src/slave/containerizer/mesos/isolators/cgroups2/cgroups2.cpp

```diff
@@ -30,6 +32,31 @@ namespace mesos {
 namespace internal {
 namespace slave {
 
+namespace cgroups2_isolator {
+
+Try<bool> supported()
```

Comment:
Could modify this to either:
1. Return `bool` and convert all `return Error`s into `return false`. Or,
2. Return `Try<Nothing>` and treat `Error()` as `false` and `Nothing()` as true.

This would make it easier for the caller, as they don't have to check two things (success and value). What's your take? @bmahler